### ISSUE TITLE
Problems are only shown occasionally

### ISSUE THREAD

**Issue by raphaelzulliger** [matangover/mypy-vscode#53]
Using `v0.2.2`, `VSCode 1.65.2`, `Python 3.8.10`

Thanks for this very useful extension! Unfortunately, it only works occasionally for me.

My project definitely does have (MyPy) issues. But they're only shown in `VSCode`'s Problems View sometimes - and I couldn't work out a pattern when they do appear and when not. Restarting `VSCode` alone does definitely not bring the problems up.

My `Output - MyPy` looks like this, e.g. when I safe a file:

```

Document saved: /home/xxxxxxx
[5] Check workspace: /home/xxxxxxx
[5] Received python path from Python extension: /home/xxxxx
[5] Running dmypy in folder /home/xxxx
/home/xxxxxx/.venv/bin/python -m mypy.dmypy --status-file /home/xxxxx/.config/Code/User/workspaceStorage/a6f4d50dfbf20faa2bd86132e36c0caa/matangover.mypy/dmypy-02a7edde1f7aba5cbbb42a190e7a92d75746676b-21556.json run --log-file /home/xxxxxxxx/.config/Code/User/workspaceStorage/a6f4d50dfbf20faa2bd86132e36c0caa/matangover.mypy/dmypy-02a7edde1f7aba5cbbb42a190e7a92d75746676b.log -- XXXXXX --show-column-numbers --no-error-summary --no-pretty --no-color-output --python-executable /home/XXXXXXX/.venv/bin/python
[5] Mypy output:

Document saved: /home/XXXXXXXX
[6] Check workspace: /home/XXXXXX
...
```

In the case above, the `/home/xxxxxxxx/.config/Code/User/workspaceStorage/a6f4d50dfbf20faa2bd86132e36c0caa/matangover.mypy/dmypy-02a7edde1f7aba5cbbb42a190e7a92d75746676b.log` file is empty, while the status file contains `{"pid": 24995, "connection_name": "/tmp/tmpz0qf3v10/dmypy.sock"}`. `dmypy` is running according to `ps ax`.

Any hints how to get it *always* working?

**Comment by raphaelzulliger** [comment]
Well... as so often, after having "talked" to someone about an issue, you get it yourself...

I just ran `mypy` in the console and it seems that my "project" does not have any issues for the MyPy "Targets" I've specified (I only specified one). Probably, since we're in early project stage, that "Target" does only import parts of the projects which do indeed not have any issues. But I then run MyPy with other "Targets" (UnitTests) and MyPy on the console showed several issues.

The "fix" to this issue is therefore to add all "entry points" to my code in the "Targets" settings option of `VSCode`. By doing so this extension properly shows 54 errors along with some hints. 

Sorry for all this fuss.
